libs/vkd3d/resource.c

```diff
@@ -5427,6 +5440,66 @@ static HRESULT d3d12_descriptor_heap_create_uav_counter_buffer(struct d3d12_desc
     return S_OK;
 }
 
+static void d3d12_descriptor_heap_update_extra_bindings(struct d3d12_descriptor_heap *descriptor_heap,
+        struct d3d12_device *device)
+{
+    const struct vkd3d_vk_device_procs *vk_procs = &device->vk_procs;
+    unsigned int i, binding_index, set_index = 0, write_count = 0;
+    VkDescriptorBufferInfo vk_buffer_info[8];
```

Comment:
Should probably have a define instead of magic value of 8.

Reply:
Added `VKD3D_BINDLESS_SET_MAX_EXTRA_BINDINGS`.